cmd/wallet-web/src/pages/LoginHandle.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div>
+    <p>Redirecting...</p>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+export default {
+  name: 'LoginRedirect',
+  props: {
+    provider: {
+      type: String,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      userinfo: null,
+    };
+  },
+  created: async function () {
+    await this.checkIfUserLoggedIn();
+    if (this.userInfo.ok) {
```

Comment:
Can't we just use a local variable here (rather than using this) ?

Reply:
local variable was throws undefined errors, thats why i used like this.  Add more to console warnings

Reply:
return userinfo from `checkIfUserLoggedIn` and,
``
let userInfo = await this.checkIfUserLoggedIn()
``